guides/support-and-on-call.md

```diff
@@ -1,4 +1,6 @@
-# Support and on call
+---
+title: Support and on call
+---
 
 We have service definitions for [GovPress hosting], [container hosting] and
```

Comment:
These don't turn into links in the HTML-ised version.
<img width="899" alt="Screenshot 2020-06-15 at 16 32 39" src="https://user-images.githubusercontent.com/579522/84676896-e5aa1080-af25-11ea-9c17-6b2e519b2a92.png">

Reply:
Oh, interesting...